components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java

```diff
@@ -873,12 +888,12 @@
                     "LEFT JOIN IDN_OAUTH2_SCOPE_BINDING AS SCOPEBINDINGS ON FILTEREDSCOPES.SCOPE_ID=SCOPEBINDINGS.SCOPE_ID";
 
     public static final String RETRIEVE_SCOPE_BY_NAME = "SELECT SCOPES.NAME, SCOPES.DISPLAY_NAME, SCOPES.DESCRIPTION, " +
-            "SCOPEBINDINGS.SCOPE_BINDING FROM IDN_OAUTH2_SCOPE AS SCOPES " +
+            "SCOPEBINDINGS.SCOPE_BINDING,SCOPEBINDINGS.BINDING_TYPE FROM IDN_OAUTH2_SCOPE AS SCOPES " +
```

Comment:
seems to be the only change between this and the oracle query is not having "AS" keyword in `... IDN_OAUTH2_SCOPE_BINDING AS SCOPEBINDINGS ...`. It should be working in all the databases. So we can remove this query.